mappings/net/minecraft/IntCache.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_604 net/minecraft/IntCache
```

Comment:
You have forgotten to put any of these classes in packages.

Reply:
where would you put this one?

Reply:
Unsure based on the class name alone. What does it do internally and where is it used?

Reply:
it seems to be some sort of allocator/cache based on ints/int arrays